app/views/candidate_interface/personal_details/immigration_right_to_work/_shared_form.html.erb

```diff
@@ -3,19 +3,17 @@
 </h1>
 
   <p class="govuk-body">
-    You’ll need to have the right to work or study for the duration of your course. You can use a visa which is due to expire before the end of your course, as long as you have the right to renew it.
-  </p>
-
-  <p class="govuk-body"> <%= govuk_link_to('Check your immigration status and find out if you need a visa',
-                    t('train_to_teach_in_england.url_citizen_visa'),
-                    target: :_blank,
-                    rel: 'noopener') %>.
+   You need a visa or immigration status that allows you to work or study in the UK while you do your teacher training. You can use a visa which is due to expire before the end of your course, as long as you have the right to renew it.
   </p>
 
   <p class="govuk-body">
     If you do not have the right to work or study, you can apply for a visa when you’ve been accepted onto a course that sponsors visas.
   </p>
 
+    <p class="govuk-body"> <%= govuk_link_to('Check if you need a UK visa',
+                    t('govuk.check_if_you_need_visa'), target: '_blank') %>.
+  </p>
+
   <%= f.govuk_radio_buttons_fieldset :right_to_work_or_study, legend: { size: 'm', text: 'Do you already have the right to work or study in the UK?' } do %>
     <%= f.govuk_radio_button :right_to_work_or_study, 'yes', label: { text: 'Yes' }, link_errors: true %>
     <%= f.govuk_radio_button :right_to_work_or_study, 'no', label: { text: 'No' } %>
```

Comment:
Indentation in this file is off

```erb
<h1 class="govuk-heading-xl">
  <%= t('page_titles.immigration_right_to_work') %>
</h1>

<p class="govuk-body">
  You need a visa or immigration status that allows you to work or study in the UK while you do your teacher training. You can use a visa which is due to expire before the end of your course, as long as you have the right to renew it.
</p>

<p class="govuk-body">
  If you do not have the right to work or study, you can apply for a visa when you’ve been accepted onto a course that sponsors visas.
</p>

<p class="govuk-body">
  <%= govuk_link_to('Check if you need a UK visa', t('govuk.check_if_you_need_visa'), target: '_blank') %>.
</p>

<%= f.govuk_radio_buttons_fieldset :right_to_work_or_study, legend: { size: 'm', text: 'Do you already have the right to work or study in the UK?' } do %>
  <%= f.govuk_radio_button :right_to_work_or_study, 'yes', label: { text: 'Yes' }, link_errors: true %>
  <%= f.govuk_radio_button :right_to_work_or_study, 'no', label: { text: 'No' } %>
<% end %>

<%= f.govuk_submit t('save_and_continue') %>
```

Reply:
Thanks - have ammened [here](https://github.com/DFE-Digital/apply-for-teacher-training/pull/8917/commits/3d676051204cab8ced4b787ccf0fb7aa9eea9992).